services/skus/credentials.go

```diff
@@ -30,6 +30,9 @@ import (
 const (
 	defaultMaxTokensPerIssuer       = 4000000 // ~1M BAT
 	defaultCohort             int16 = 1
+
+	// maxTLV2ActiveDailyItemCreds specifies the number of credentials an item is allowed to have in a given day.
+	maxTLV2ActiveDailyItemCreds = 10
```

Comment:
Is this something you would want to eventually pass in as an env var?

Reply:
If you pass as an env var through the ops sealed secrets or configmap you can update these types of values without needing to rebuild the image. Just some food for thought.

Reply:
> Is this something you would want to eventually pass in as an env var?

Thanks for asking.

Not at this point. This will eventually be made per product per subscription, and might not even live here. Hard-coding it for now because that's what the dynamic value in another service has been.